build/settings.js

```diff
@@ -38,7 +38,8 @@ settings = {
   remoteUrl: 'https://api.resin.io',
   dashboardUrl: 'https://dashboard.resin.io',
   dataDirectory: path.join(userHome, '.resin'),
-  imageCacheTime: 1 * 1000 * 60 * 60 * 24 * 7
+  imageCacheTime: 1 * 1000 * 60 * 60 * 24 * 7,
+  tokenValidityTime: 1 * 1000 * 60 * 60
```

Comment:
Sure, that's why it's a soft setting that can be customised by the user, rather than a hardcoded "only source of truth" value, although maybe worth to add a comment to clarify?